spec/factories/in_person_enrollments.rb

```diff
@@ -3,6 +3,14 @@
     user { association :user, :signed_up }
     profile { association :profile, user: user }
     unique_id { Faker::Number.hexadecimal(digits: 18) }
+
+    trait :establishing do
+      after :build do |enrollment|
+        enrollment.status = :establishing
+        enrollment.enrollment_code = Faker::Number.number(digits: 16)
+      end
+    end
+
```

Comment:
This change broke one of the existing tests:

```
Failures:
  1) Api::Verify::PasswordConfirmController#create when the user is signed in and submits the password with in person profile leaves the enrollment in establishing when no enrollment code is returned
     Failure/Error: expect(enrollment.enrollment_code).to be_nil
       expected: nil
            got: "3475987911326073"
     # ./spec/controllers/api/verify/password_confirm_controller_spec.rb:141:in `block (5 levels) in <top (required)>'
```

Reply:
Woops! Fixed in [518bcda](https://github.com/18F/identity-idp/pull/6743/commits/518bcdaad12a1efcf39cb93e65b79566d2c0e295)